src/Compilers/Core/Portable/Compilation.EmitStream.cs

```diff
@@ -130,14 +131,30 @@ internal void Close()
                 {
                     RoslynDebug.Assert(_strongNameProvider != null);
 
+                    var fileSystem = _strongNameProvider.FileSystem;
+                    var tempDir = fileSystem.GetSigningTempPath();
+                    if (tempDir is null)
+                    {
+                        diagnostics.Add(StrongNameKeys.GetError(
+                            _strongNameKeys.KeyFilePath,
+                            _strongNameKeys.KeyContainer,
+                            new CodeAnalysisResourcesLocalizableErrorArgument(nameof(CodeAnalysisResources.SigningTempPathUnavailable)),
+                            messageProvider));
+                        return null;
+                    }
+
+                    _stream = _emitStreamProvider.GetOrCreateStream(diagnostics);
```

Comment:
>_stream = _emitStreamProvider.GetOrCreateStream(diagnostics); [](http://example.com/codeflow?start=20&length=61)

It's not clear what the relationship is between this `_stream` and the `tempStream` we're returning below. We don't appear to be using `_stream` after creating it here.